indico/modules/events/schemas.py

```diff
@@ -16,4 +20,14 @@ class EventPermissionsSchema(mm.Schema):
     acl_entries = PrincipalPermissionList(EventPrincipal, all_permissions=True)
 
 
+class EventDetailsSchema(mm.ModelSchema):
+
+    class Meta:
+        model = Event
+        fields = ('id', 'category', 'title', 'start_dt', 'end_dt')
+
+    category = fields.Nested(CategorySchema)
+
+
 event_permissions_schema = EventPermissionsSchema()
+event_details_schema = EventDetailsSchema()
```

Comment:
let's not add new global schema instances (we pretty much stopped doing it for most newer schemas). better to just instantiate it when we need it